addons/crm_claim/migrations/9.0.1.0/pre-migration.py

```diff
@@ -17,7 +17,12 @@
 ]
 
 
-@openupgrade.migrate()
-def migrate(cr, version):
-    openupgrade.rename_columns(cr, column_renames)
-    openupgrade.rename_tables(cr, table_renames)
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.rename_columns(env.cr, column_renames)
+    openupgrade.rename_tables(env.cr, table_renames)
+    if openupgrade.is_module_installed(env.cr, 'website_crm_claim'):
```

Comment:
I'm talking about leaving this as it was before this PR. You don't need to do this with the renaming of the module.